db/migrate/20230206143214_create_webhooks.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class CreateWebhooks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :webhooks, id: :uuid do |t|
+      t.references :organization, index: true
+
+      t.uuid :object_id, null: false
+      t.string :object_type, null: false
```

Comment:
No index?

Reply:
no real need here, it's just for information more than usage